src/gui/sdlmain.cpp

```diff
@@ -3838,14 +4092,32 @@ bool GFX_Events()
 									CPU_Disable_SkipAutoAdjust();
 								}
 
-								/* Now poke a "release ALT" command into the keyboard buffer
-								 * we have to do this, otherwise ALT will 'stick' and cause
-								 * problems with the app running in the DOSBox.
+								/* Now poke a
+								 * "release ALT"
+								 * command into
+								 * the keyboard
+								 * buffer we have
+								 * to do this,
+								 * otherwise ALT
+								 * will 'stick'
+								 * and cause
+								 * problems with
+								 * the app running
+								 * in the DOSBox.
```

Comment:
Was this the clang formatters doing? I thought it uses a different maxwidth for comments.

Reply:
Yes - probably it has a limit how far from the start of the line can the comment reach - and here the indentation level clearly got out of control. I'm not even sure why it decided to rework these lines.

I have taken plain sdlmail.cpp from main branch and reapplied all my changes there.